js/restaurant.js

```diff
@@ -324,6 +324,12 @@ function showHelp() {
   var help = level.help || "";
   var examples = level.examples ||[];
   var selector = level.selector || "";
+  var solutions = []
+  if (progress.guessHistory[currentLevel]) {
+    if (progress.guessHistory[currentLevel].solutions) {
+      solutions = progress.guessHistory[currentLevel].solutions;
+    }
+  }
```

Comment:
Get `solutions` from `progress`. If the `progress` exists it will populate its `solutions`. But additional check is needed because level progress might exist without `solutions` and in that case `progress.guessHistory[currentLevel].solutions` will return `undefined` which is obviously not iterable